### ISSUE TITLE
self is not defined

### ISSUE THREAD

**Issue by montogeek** [lucasfeliciano/fetch-everywhere#2]
Hi!

I am transpiling my library with Rollup, this is the compiled file:

``` js
var fetchNpmBrowserify = __commonjs(function (module, exports, global) {
// the whatwg-fetch polyfill installs the fetch() function
// on the global object (window or self)
//
// Return that as the export for use in Webpack, Browserify etc.

var globalObj = self || __commonjs_global.self || __commonjs_global;
module.exports = globalObj.fetch.bind(globalObj);
});
```

Then, another project that uses my library is running its tests in a Node environment, getting the self is not defined error:

``` js
var globalObj = self || __commonjs_global.self || __commonjs_global;
                ^

ReferenceError: self is not defined
```

**Comment by jtuchsen** [comment]
I ran into this issue with React Native as well. I've submitted a pull request to fix the issue by adding a typeof check around the self variable.

**Comment by lucasfeliciano** [comment]
This should be fixed with #3 :)

Let me know if this error persists
